// ngx-markdown-editor/esm2020/lib/md-editor.types.mjs
export const DEFAULT_LOCALES = {
    en: {
        Buttons: {
            Bold: { title: 'Bold', initText: 'Bold Text' },
            Italic: { title: 'Italic', initText: 'Italic Text' },
            Heading: { title: 'Heading' },
            Reference: { title: 'Reference' },
            Link: { title: 'Link' },
            Image: { title: 'Image' },
            UnorderedList: { title: 'Unordered List', initText: '' },
            OrderedList: { title: 'Ordered List', initText: '' },
            CodeBlock: { title: 'Code Block', initText: 'Source Code' },
            ShowPreview: { title: 'Show Preview' },
            HidePreview: { title: 'Hide Preview' },
            Fullscreen: { title: 'Fullscreen' }
        },
        Upload: {
            Drag: 'Drag it here',
            Uploading: 'Uploading'
        }
    }
};
export const DEFAULT_ICONS = {
    fontAwesome4: {
        Bold: { fontClass: 'fa fa-bold' },
        Italic: { fontClass: 'fa fa-italic' },
        Heading: { fontClass: 'fa fa-header' },
        Reference: { fontClass: 'fa fa-quote-left' },
        Link: { fontClass: 'fa fa-link' },
        Image: { fontClass: 'fa fa-image' },
        UnorderedList: { fontClass: 'fa fa-list-ul' },
        OrderedList: { fontClass: 'fa fa-list-ol' },
        CodeBlock: { fontClass: 'fa fa-file-code-o' },
        ShowPreview: { fontClass: 'fa fa-eye' },
        HidePreview: { fontClass: 'fa fa-eye-slash' },
        FullScreen: { fontClass: 'fa fa-arrows-alt' },
        CheckBox_UnChecked: { fontClass: 'fa fa-square-o' },
        CheckBox_Checked: { fontClass: 'fa fa-check-square' },
    },
    fontAwesome5: {
        Bold: { fontClass: 'fas fa-bold' },
        Italic: { fontClass: 'fas fa-italic' },
        Heading: { fontClass: 'fas fa-heading' },
        Reference: { fontClass: 'fas fa-quote-left' },
        Link: { fontClass: 'fas fa-link' },
        Image: { fontClass: 'fas fa-image' },
        UnorderedList: { fontClass: 'fas fa-list-ul' },
        OrderedList: { fontClass: 'fas fa-list-ol' },
        CodeBlock: { fontClass: 'fas fa-file-code' },
        ShowPreview: { fontClass: 'fas fa-eye' },
        HidePreview: { fontClass: 'fas fa-eye-slash' },
        FullScreen: { fontClass: 'fas fa-compress' },
        CheckBox_UnChecked: { fontClass: 'far fa-square' },
        CheckBox_Checked: { fontClass: 'fas fa-check-square' }
    },
    fontAwesome6: {
        Bold: { fontClass: 'fa-solid fa-bold' },
        Italic: { fontClass: 'fa-solid fa-italic' },
        Heading: { fontClass: 'fa-solid fa-heading' },
        Reference: { fontClass: 'fa-solid fa-quote-left' },
        Link: { fontClass: 'fa-solid fa-link' },
        Image: { fontClass: 'fa-solid fa-image' },
        UnorderedList: { fontClass: 'fa-solid fa-list-ul' },
        OrderedList: { fontClass: 'fa-solid fa-list-ol' },
        CodeBlock: { fontClass: 'fa-solid fa-file-code' },
        ShowPreview: { fontClass: 'fa-solid fa-eye' },
        HidePreview: { fontClass: 'fa-solid fa-eye-slash' },
        FullScreen: { fontClass: 'fa-solid fa-maximize' },
        CheckBox_UnChecked: { fontClass: 'fa-regular fa-square' },
        CheckBox_Checked: { fontClass: 'fa-solid fa-check-square' }
    }
};
